include/aws/common/math.inl

```diff
@@ -52,6 +52,10 @@ AWS_EXTERN_C_BEGIN
 #    endif /*  AWS_HAVE_GCC_OVERFLOW_MATH_EXTENSIONS */
 #endif     /*  defined(AWS_HAVE_GCC_OVERFLOW_MATH_EXTENSIONS) && (defined(__clang__) || !defined(__cplusplus)) */
 
+#if defined(__clang__) || defined(__GNUC__)
```

Comment:
Probably just my lack of familiarity with the structure here--but I was wondering how the paths that include math.gcc_overflow.inl and math.gcc_x64_asm.inl get the clz/ctz functionality since it looks like it's only in math.gcc_builtin.inl now?  (Or I guess if they still need it?)